snyk/models.py

```diff
@@ -162,7 +162,9 @@ def notification_settings(self):
 
     # https://snyk.docs.apiary.io/#reference/organisations/the-snyk-organisation-for-a-request/invite-users
     def invite(self, email: str, admin: bool = False):
```

Comment:
If we're already using type hints for arguments then how about explicitly hinting the method's return type as well?